importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/TagImporterFactory.java

```diff
@@ -29,7 +26,8 @@ public class TagImporterFactory extends ImporterFactory
             "json", "htm", "html", "xhtml", "yaml", // markup languages
             "java", // Java
             "clj", "kt", "scala", // JVM languages
-            "js", // Java script
+            "js", // JavaScript
+            "ts", // TypeScript

```

Comment:
Secretly sneaked Typescript in there? :-)

Did you add that to the handbook too?

Reply:
✅